Client/sources/components/fileUpload/filePicker.tsx

```diff
@@ -71,10 +78,13 @@ const handleFileUpload = async (
   setFileToUpload: React.Dispatch<File | undefined>,
   inputRef: React.RefObject<HTMLInputElement>,
   fileUploadProgress: (event: any) => void,
-  setProgress: React.Dispatch<number>
+  setProgress: React.Dispatch<number>,
+  currentDir: IFolder
```

Comment:
pass path here because we are using currentDir.path. No need to pass this object